TWLight/ezproxy/views.py

```diff
@@ -32,6 +32,9 @@ def get(request, url=None, token=None):
         if not username:
             raise SuspiciousOperation("Missing Editor username.")
 
+        if request.user.editor.wp_bundle_eligible:
```

Comment:
Should this be checked with this bool or by looking for the user's Bundle authorization?

Reply:
Yeah, it should check the bundle auth. Those didn't exist when this was written.